### PR TITLE
UCT/RC/DC: Do not translate TMH to BE

### PR DESCRIPTION
**Pull request by brminich**
Remove translation to BE during TMH send and to LE when receive it.

### CODE REVIEW

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6603/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6615/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6620/ for details (Mellanox internal link).

File: src/uct/ib/rc/verbs/rc_verbs_common.c [hunk 1]
```diff
@@ -189,3 +189,11 @@ ucs_status_t uct_rc_verbs_wc_to_ucs_status(enum ibv_wc_status status)
         return UCS_ERR_IO_ERROR;
     }
 }
+
+void uct_rc_verbs_common_packet_dump(uct_base_iface_t *iface, uct_am_trace_type_t type,
```
Comment (hoopoepg) [inline src/uct/ib/rc/verbs/rc_verbs_common.c]: minor: may be move it to header & mark as inline?

File: src/uct/ib/rc/accel/rc_mlx5_common.h [hunk 2]
```diff
@@ -114,6 +114,27 @@ typedef struct uct_rc_mlx5_cmd_wq {
...
+    tmh->tag     = tag;
+}
+
+#  define UCT_RC_MLX5_IFACE_GET_TM_BCOPY_DESC(_iface, _mp, _desc, _tag, _app_ctx, \
+                                              _pack_cb, _arg, _length) \
+       { \
+           void *hdr; \
+           UCT_RC_IFACE_GET_TX_DESC(_iface, _mp, _desc) \
+           (_desc)->super.handler = (uct_rc_send_handler_t)ucs_mpool_put; \
+           hdr = _desc + 1; \
```
Comment (yosefe) [inline src/uct/ib/rc/accel/rc_mlx5_common.h]: pls add () around _desc

File: src/uct/ib/rc/verbs/rc_verbs_common.c (hunk 1 above)
Comment (yosefe) [reply src/uct/ib/rc/verbs/rc_verbs_common.c]: slow path function - i'd say it's not needed

**Comment by mellanox-github** [comment]
Test FAILed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6630/ for details (Mellanox internal link).

**Comment by mellanox-github** [comment]
Test PASSed.
See http://hpc-master.lab.mtl.com:8080/job/hpc-ucx-pr/6632/ for details (Mellanox internal link).

**Pull request merged**